feature/trainer/addptsession/src/main/java/co/kr/tnt/trainer/addptsession/AddPtSessionScreen.kt

```diff
@@ -500,19 +481,15 @@ private fun Memo(
     isWarning: Boolean,
     onValueChanged: (String) -> Unit,
 ) {
-    Text(
-        text = stringResource(R.string.write_memo),
-        style = TnTTheme.typography.body1Bold,
-        color = TnTTheme.colors.neutralColors.Neutral900,
-    )
-    Spacer(modifier = Modifier.height(8.dp))
-    TnTOutlinedTextField(
+    TnTLabeledTextField(
```

Comment:
PT 세션 추가 화면의 `메모하기` 부분에 placeholder가 빠져있습니다!
```suggestion
    TnTLabeledTextField(
        placeholder = "PT 수업에서 기억해야 할 것을 메모해보세요",
```